ApiEverest/AppServices/Services/CostumerAppService.cs

```diff
@@ -0,0 +1,33 @@
+﻿using DomainModels.Entities;
+
+namespace AppServices.Services
+{
+    public class CustomerAppService : ICustomerAppService
+    {
+        private readonly ICustomerAppService _customerAppService;
```

Comment:
```suggestion
        private readonly ICustomerService _customerService;
```
Deveria estar chamando a camada de servico